__tests__/__unit__/dataset/DatasetTree.unit.test.ts

```diff
@@ -0,0 +1,1176 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import * as vscode from "vscode";
+import * as globals from "../../../src/globals";
+import * as fs from "fs";
+import * as zowe from "@zowe/cli";
+import { DatasetTree } from "../../../src/dataset/DatasetTree";
+import { ZoweDatasetNode } from "../../../src/dataset/ZoweDatasetNode";
+import * as utils from "../../../src/utils";
+import { Profiles } from "../../../src/Profiles";
+import { getIconByNode } from "../../../src/generators/icons";
+import {
+    createInstanceOfProfile,
+    createIProfile,
+    createISession, createISessionWithoutCredentials, createQuickPickContent,
+    createTreeView, createWorkspaceConfiguration
+} from "../../../__mocks__/mockCreators/shared";
+import { createDatasetSessionNode } from "../../../__mocks__/mockCreators/datasets";
+import { bindMvsApi, createMvsApi } from "../../../__mocks__/mockCreators/api";
+
+jest.mock("fs");
+jest.mock("util");
+
+function createGlobalMocks() {
+    const isTheia = jest.fn();
```

Comment:
Same comment as I did in util test file. I guess this is a common pattern. Let's leave it in here for this PR and make a followup that will fix it across the codebase.